### DISCUSSION TITLE
Can't find ImportHyperLynx

### DISCUSSION THREAD

**Discussion by tabarr** [thliebig/openEMS-Project#82] [Q&A Usage] [answered]
Trying to follow the  "Importing with hyp2mat" tutorial. I know my setup works because I can get the simple_patch_antenna.m file to run. But when I try to do a HyperLynx import I get a ImportHyperLynx is not defined message. I also do not see a ImportHyperLynx.m file in the matlab directory. I am using the latest windows version of openEMS.

**Comment by thliebig** [comment, marked as answer]
Hm, that might have gotten lost on the transition to the newer MSVC based builds?
Maybe you can check if this is all still available in the older [v0.0.35-74](https://github.com/thliebig/openEMS-Project/releases/tag/v0.0.35-74) build?

**Reply by tabarr** [reply]
I did find a hyp2mat directory in the older build. Am going to try copying that folder into the newer build.

**Reply by tabarr** [reply]
Copying the files seems to work, after I added the \hyp2mat\matlab path to the Octave statrtup.m and octavcerc files. Getting some syntax errors that I need to investigate, but I do get a valid layer setup.

**Reply by tabarr** [reply]
Got a bit further. But the hyp2mat example has the following calls that are not recognized:
unit = GetUnits(CSX);
substrate_epr = GetEpsilon(CSX);

**Reply by tabarr** [reply]
Altium Hyperlynx export seems  to have some issues so I am going to have to try a different method.

**Comment by daft-develop** [comment]
I was able to get the, "Importing with hyp2mat" tutorial working today (with v0.0.36) after making the following adjustments:

1. The hyp2mat folder appears to be missing on all the windows releases past v0.0.35-74.
    - I downloaded the pre-built binary of hyp2mat as suggested in the hyp2mat repo - README.md commit b15f749 (master) of koendv/hyp2mat suggests [this pre-compiled binary](http://www.kdvelectronics.eu/hyp2mat/hyp2mat-win32-current.zip)
    - I tested the bundled binary version from v0.0.35-74 (manual copy only the hyp2mat folder to v0.0.36) and it worked as well
2. Added additional paths in Octave for the missing functions:
    - Standard folder openEMS\matlab
    - Additional folder openEMS\hyp2mat\matlab for the ImportHyperLynx function
    - Additional folder openEMS\hyp2mat\eagle\tutorial for the GetUnits and GetEpsilon functions
